### PR TITLE
[Bug]: fix titles of pages (dashboard, apps-settings, users-settings) #43327

### PR DESCRIPTION
**Pull request by arublov**
* Resolves: #43327

## Summary

1. Remove pageTitle from Dashboard (remove duplication)
2. Changed on Settings/App from (Apps - Apps - Nextcloud) to (Settings - Apps - Nextcloud)
3. Changed on Users from (Users - Users - Nextcloud) to (Settings - Users - Nextcloud)

### CODE REVIEW

**Review by JuliaKirschenheuter — APPROVED** [review]

**Review by emoral435 — CHANGES_REQUESTED** [review]
Everything looks good, but this would introduce a new regression and an accessibility issue (regression in question being #42267)

image

The heading is now not descriptive enough. I personally think this is fine - but this is not up to me, as the checker for accessibility described this as being a breaking feature, in terms of accessibility, so it needs to be something different. Everything else looks good, so once that is fixed everything should be good to go :)

File: core/templates/layout.user.php [hunk 1]
```diff
@@ -19,9 +19,9 @@
 		<title>
 			<?php
 				p(!empty($_['pageTitle'])?$_['pageTitle'].' - ':'');
-p(!empty($_['application'])?$_['application'].' - ':'');
-p($theme->getTitle());
-?>
+				p(!empty($_['application'])?$_['application'].' - ':'');
```
[the remark this answers is not in the document]
Comment (emoral435) [reply core/templates/layout.user.php:22]: Yes, that would fix the issue (both the current, and the original). cc @arublov would you like to finish this, or would you want me to pick it up? either is fine :) Thank you for the contribution 💙

[the remark this answers is not in the document]
Comment (arublov) [reply core/templates/layout.user.php:22]: @emoral435 , yes, of course I'd like to finish it! 

I actually brought this up on a call with @ChristophWurst today, so yes, I'd like to see it through :)
I'll get started on it right away.

Thanks for your comment :)

**Comment by arublov** [comment]
@emoral435 Done ✅

**Comment by arublov** [comment]
And I see this PR into stable28, should I change this? 
Or leave it and it's ok? 
Because this is an old PR and I was creating it from stable28, now I yes, understand that it has to be created from a master.

**Comment by szaimen** [comment]
Screenshot_20240214_162520_Brave
👀

**Comment by arublov** [comment]
> Screenshot_20240214_162520_Brave 👀

yeap, I asked that above, okay, let's close this PR and create a new, right?

> Screenshot_20240214_162520_Brave 👀 Can we please target master first?

yeap, I asked that above, okay, let's close this PR and create a new, right?

**Comment by emoral435** [comment]
@arublov Yep! Just tag us as reviewers and we will approve and get it merged :) 💙

**Pull request closed without merging**